### ISSUE TITLE
About train KITTI

### ISSUE THREAD

**Issue by heuzxy** [JarvisLee0423/GREAT-Stereo#3]
when we training on kitti, we found  don't need 50k steps. the validation is smooth. And the learning rate is 0.002, size=320x736.

**Comment by JarvisLee0423** [comment]
Sorry, I do not fully understand what your question is.

If you mean the submission ckpt of the kitti benchmark, our training strictly follows the pipeline described in the paper, and we save the ckpt at a frequency of 10000. However, for the final submission ckpt, we will pick up the best results from all the saved ckpts. It always seems to be the 40k ckpt or the 50k ckpt.

And more precisely, in our training pipeline, the learning rate is 0.0002.
